whats-new-panel.json

```diff
@@ -73,7 +73,7 @@
         "string_id": "cfr-whatsnew-pip-cta"
       }
     },
-    "targeting": "firefoxVersion >= 76",
+    "targeting": "firefoxVersion >= 76 && (firefoxVersion < 77 || locale != \"en-GB\")",
```

Comment:
This is meant to push out message #3 in `en-GB` locale to keep 3 messages in the panel.